services/src/main/java/org/keycloak/services/resources/RealmsResource.java

```diff
@@ -115,6 +119,41 @@ public Object getProtocol(final @PathParam("realm") String name,
         return endpoint;
     }
 
+    /**
+     * Returns a temporary redirect to the client url configured for the given {@code clientId} in the given {@code realmName}.
+     * <p>
+     * This allows a client to refer to other clients just by their client id in URLs, will then redirect users to the actual client url.
+     * The client url is derived according to the rules of the base url in the client configuration.
+     * </p>
+     *
+     * @param realmName
+     * @param clientId
+     * @return
+     * @since 1.9
+     */
+    @GET
+    @Path("{realm}/clients/{client_id}/redirect")
```

Comment:
This is the same path as the client-registration service is registered at. We either need a different path for this or move the client-registration. With regards to having a clients landing page like we discussed in mailing list it makes more sense to move client registration service, but that means changing endpoints that are already included in past releases. I'll figure out what to do here.